docs/build_and_load.md

```diff
@@ -339,6 +339,166 @@ file contains 6 columns of information:
 > region.
 
+### Annotate Fastas
+The annotate-fasta command has two goals.  The first is to copy the fastas to a new file
+whose name is changed to be <sampleName>.fa.  The second is to add a sampleName to the id lines of the fasta file.
+**These annotated fasta files should be used as input to both the agc-compress step and the align-assemblies step.**
+This ensures consistent sample names across the pipeline.
+
+For the first goal, a file originally named Zm-CML52-NAM-1.0.fa would be copied to a new file named CML52.fa, based
+on the user supplying a keyfile with "CML52" as the sample name for this fasta.  The reason
+for this change is the AGC compression tool stores the file name (minus extension) as the sample name
+when creating the compressed file.  This keeps the AGC sample name consistent with sample
+names that are stored in the VCF files.  With consistent sample names, sequence and variants may be associated.
+
+For the second goal:  As of the current date of this document, the return methods of
+AGC will not keep track of sample IDs when returning
+sequence information from the compressed file **unless you explicitly
+state the sample information in the header lines of the FASTA files
+you wish to compress for the PHGv2 databases**. To explain this
+further, let's imagine that we have two FASTA files: `LineA.fa` and
+`LineB.fa`. These files contain information for only chromosome 1 and
+have a simple header that denotes this sequence name (e.g. `chr1`):
+
+```
+$ head LineA.fa
+>chr1
+ATGCGTACGCGCACCG
+
+$ head LineB.fa
+>chr1
+ATGCGTTCGCCTTCCG
+```
+
+After we compress this information using the `agc-compress` command,
+we will have a file called `assemblies.agc`. PHGv2 leverages this
+compressed sequence file when creating VCF data (_see
+the [**"Create VCF files"**](#create-vcf-files) section for further
+details_). The issue arrives when we query the compressed data using
+[AGC's `getctg`](https://github.com/refresh-bio/agc#extract-contigs-from-the-archive)
+command. When we query this hypothetical archive, we will get the
+following information:
+
+```
+$ agc getctg assemblies.agc chr1@LineA chr1@LineB > queries.fa
+
+$ head queries.fa
+>chr1
+ATGCGTACGCGCACCG
+>chr1
+ATGCGTTCGCCTTCCG
+```
+
+As you can see from the above hypothetical output, we now have no
+means to efficiently track where each sequence is coming from due
+to the lack of header information from our prior FASTA files. We
+can remedy this by adding sample information to the headers:
+
+```
+$ head LineA.fa
+>chr1 sampleName=LineA
+ATGCGTACGCGCACCG
+
+$ head LineB.fa
+>chr1 sampleName=LineB
+ATGCGTTCGCCTTCCG
+```
+
+Now, when we compress and query using AGC, we get enhanced sample ID
+tracking:
+
+```
+$ ./agc getctg assemblies_new_headers.agc chr1@LineA chr1@LineB > queries.fa
+
+$ head queries.fa
+>chr1 sampleName=LineA
+ATGCGTACGCGCACCG
+>chr1 sampleName=LineB
+ATGCGTTCGCCTTCCG
+```
+
+While we can either manually modify the header lines of our FASTA
```

Comment:
"either" is a typo?

Reply:
yup, looks like it.  Will remove "either".